packages/extension/src/view/devtools/components/cookies/cookiesListing/cookieTableContainer/editableComponents/editableCheckBox.tsx

```diff
@@ -0,0 +1,126 @@
+
+/**
+ * Internal dependencies.
+ */
+import { useCookieStore } from '../../../../../stateProviders/syncCookieStore';
+import classNames from 'classnames';
+import type { InfoType } from '@ps-analysis-tool/design-system';
+import { noop } from '@ps-analysis-tool/common';
+
+interface EditableCheckBoxInputProps {
+  info: InfoType;
+  changedKey: string;
+  cookieKey?: string | null;
+  rowHighlighter?: (value: boolean, cookieKey: string) => void;
+}
+
+const EditableCheckBoxInput = ({
+  info,
+  changedKey,
+  cookieKey,
+  rowHighlighter = noop,
+}: EditableCheckBoxInputProps) => {
+  const [editing, setEditing] = useState(false);
+  const divRef = useRef<HTMLInputElement>(null);
+
+  const [localValue, setLocalValue] = useState<boolean>(info as boolean);
+  const { modifyCookie } = useCookieStore(({ actions }) => ({
+    modifyCookie: actions.modifyCookie,
+  }));
+
+  const handleDoubleClick = useCallback(
+    (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      if (event.detail === 2) {
+        setEditing(true);
+      }
+    },
+    []
+  );
+
+  useEffect(() => {
+    if (editing) {
+      divRef.current?.focus();
+    }
+  }, [editing]);
+
+  const handleClickOutside = useCallback(
+    async (event: MouseEvent) => {
+      if (divRef.current && !divRef.current.contains(event.target as Node)) {
+        setEditing(false);
+        if (localValue !== info && cookieKey) {
+          const result = await modifyCookie(
+            cookieKey,
+            changedKey,
+            localValue,
+            null
+          );
+          if (!result) {
+            rowHighlighter(true, cookieKey);
+            setLocalValue(info as boolean);
+          }
+        }
+      }
+    },
+    [info, changedKey, localValue, modifyCookie, cookieKey, rowHighlighter]
+  );
+
+  useEffect(() => {
+    document.addEventListener('mousedown', handleClickOutside);
+    return () => {
+      // Unbind the event listener on clean up
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, [handleClickOutside]);
+
+  useEffect(() => {
+    setLocalValue(info as boolean);
+    setEditing(false);
+  }, [info]);
+
+  return (
+    <div
+      className="h-full w-full flex items-center justify-center"
+      onClick={(event) => handleDoubleClick(event)}
+      onKeyDown={(e) => e.stopPropagation()}
+    >
+      {editing ? (
+        <input
+          ref={divRef}
+          type="checkbox"
+          checked={localValue}
+          className={classNames(
+            'accent-royal-blue dark:accent-orange-400 w-3 h-3 dark:bg-outer-space dark:min-h-0 dark:min-w-0 dark:h-[13px] dark:w-[13px]',
+            {
+              'dark:appearance-none dark:text-manatee dark:border dark:rounded-[3px]':
+                !localValue,
+            }
+          )}
+          onChange={(e) => setLocalValue(e.target.checked)}
+        />
+      ) : (
+        <p>{info ? <span className="font-serif">✓</span> : ''}</p>
```

Comment:
```suggestion
        <span className="font-serif">{info ?'✓': ''}</span>
```

Reply:
Fixed in [6cf988b](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/278/commits/6cf988bc6ebd111c54a46190eb19cbbf2dbba6b4)